libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -229,6 +250,25 @@ void AP_MotorsMulticopter::output()
     output_boost_throttle();
 };
 
+// limit roll pitch and yaw inputs and set limit flags
+void AP_MotorsMulticopter::limit_input_roll_pitch_yaw()
+{
+    if (fabs(_roll_in) > _lim_rll && _lim_rll > 0.0f) {
+        _roll_in = constrain_float(_roll_in, -_lim_rll, _lim_rll);
+        limit.roll = true;
+        limit.pitch = true;
```

Comment:
I am guessing that limiting pitch here too is on purpose, is that correct?

Reply:
I removed this code